app/README.md

```diff
@@ -43,3 +43,44 @@ This React-Redux project follows the overall [Redux style guide](https://redux.j
 
 ### Testing guiding principles
 [The more your tests resemble the way your software is used, the more confidence they can give you.](https://testing-library.com/docs/guiding-principles)
+
+
+# App Dependencies
+
+
+## Redux ToolKit
+`Redux ToolKit` is a package used to handle the whole application state. Its module `RTK-Query` is also used to handle API calls.
+We made the choice of handling the state in 2 different ways : 
+
+
+### - Using `RTK-Query`'s cached requests responses
+The whole API schema is automatically generated by `tox` python module. This allows us to generate all the available API routes with `rtk-incubator` and thus it gives us access to a whole load of `Hooks` used to request data. Those hooks are wired to a redux `store` under the hood and the whole cache system is handled by `RTK-Query` itself. See `https://redux-toolkit.js.org/rtk-query/overview` for more information.

```

Comment:
the generator is called `rtk-query-codegen` https://github.com/rtk-incubator/rtk-query-codegen . it will be merged in the main rtk repo soonish.